### PR TITLE
Update coord limit torques

### PR DESCRIPTION
**Pull request by Lars-DHondt-KUL**
## Description

- Read coefficients of coordinate limit torques from a text file (.csv) instead of from a matlab function. The file can be changed via a setting.
- Added setting to scale the magnitude of all limit torques.
- Added setting to set the neutral position of a coordinate stiffness to a nonzero angle.

## Motivation and Context

- These make the limit torques easier to modify, especially for models with different coordinate definitions.

## How Has This Been Tested?

./Tests/test_PredSim.m

## Suggested tests for reviewers

./Tests/test_PredSim.m

### CODE REVIEW

File: PreProcessing/get_passive_moment_info.m
Comment (VDB-Bram) [inline PreProcessing/get_passive_moment_info.m:1]: this has no direct relation to the changes made, but wondering why L125-127 is there in the updated file. That elseif statement will never be true, right?

File: README.md [hunk 1]
```diff
@@ -276,9 +276,18 @@ This code can automatically convert an OpenSim model to the external function us
...
 - **S.subject.set_stiffness_coefficient_selected_dofs**: 
-	- stiffness coefficient can be specified here for each coordinate individually. For example, S.subject.set_stiffness_coefficient_selected_dofs = {{'hip_flexion_l','hip_flexion_r'},0.012,{'knee_angle_l'},0.011} will put stiffness coefficient of both hip flexions to 0.012 Nm/rad and that of knee angle left to 0.011 Nm/rad. If not defined here for a particular coordinate, S.subject.damping_coefficient_all_dofs will be used for that coordinate. Default is empty.
+	- stiffness coefficient can be specified here for each coordinate individually. For example, S.subject.set_stiffness_coefficient_selected_dofs = {{'hip_flexion_l','hip_flexion_r'},0.012,{'knee_angle_l'},0.011} will put stiffness coefficient of both hip flexions to 0.012 Nm/rad and that of knee angle left to 0.011 Nm/rad. If not defined here for a particular coordinate, S.subject.stiffness_coefficient_all_dofs will be used for that coordinate. Default is empty.
+- **S.subject.set_stiffness_offset_selected_dofs**: 
+	- position where moment of linear stiffness is zero can be specified here for each coordinate individually. Default is empty.
```
Comment (VDB-Bram) [inline README.md:281]: way how to define this is missing; is it similar to e.g. S.subject.set_damping_coefficient_selected_dofs?

File: README.md [hunk 2]
```diff
@@ -276,9 +276,18 @@ This code can automatically convert an OpenSim model to the external function us
...
+- **S.subject.default_coord_lim_torq_coeff**:
+	- file with default coefficients for coordinate limit torques. Default is *'default_coord_lim_torq_coeff.csv'* [char].
+	The provided file should be compatible with [`readtable`](https://mathworks.com/help/matlab/ref/readtable.html). The table should contain a column with coordinate names (header: name), 4 columns with stiffness coefficients, (headers: K_1, K_2, K_3, K_4), and 2 columns with offset coefficients (headers: theta_1, theta_2).
+	Limit torques are calculated in function of coordinate value q as: `Tau = K(1)*exp(K(2)*(q-theta(2))) + K(3)*exp(K(4)*(q-theta(1)))`.
+	Default coefficients are taken from *Anderson III, Frank Clayton. A dynamic optimization solution for a complete cycle of normal gait. The University of Texas at Austin, 1999.*
+- **S.subject.scale_default_coord_lim_torq**:
+	- scale factor for the amplitude of all coordinate limit torques. Default is empty [double].
```
Comment (VDB-Bram) [inline README.md:288]: way how to define this is missing; is it similar to e.g. S.subject.set_damping_coefficient_selected_dofs?

File: README.md [hunk 3]
```diff
@@ -276,9 +276,18 @@ This code can automatically convert an OpenSim model to the external function us
...
 - **S.subject.set_limit_torque_coefficients_selected_dofs**: 
-	- Default values of coordinate limit torques are defined in the function [get_default_coord_limit_torque_coefficients.m](https://github.com/KULeuvenNeuromechanics/PredSim/blob/master/PreProcessing/get_default_coord_limit_torque_coefficients.m). If values other than these are to be used, they can be specified here.
+	- Set limit torque coefficients for a coordinate. Default is empty [cell array] with pattern {coordinate name(s) [char, cell array of chars], K [4x1 double], theta [2x1 double]}.
```
Comment (VDB-Bram) [inline README.md:290]: Additional info needed on how this has priority over default_coord_lim_torq?

File: PreProcessing/get_passive_moment_info.m
Comment (Lars-DHondt-KUL) [reply PreProcessing/get_passive_moment_info.m:1]: indeed, I removed it

File: README.md (hunk 3 above)
Comment (Lars-DHondt-KUL) [reply README.md:290]: I added that these coefficients take priority.

**Review by VDB-Bram — APPROVED** [review]

**Pull request merged**